src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((value) => value.trim())
+    .filter((value) => value !== '')
+    .reduce((prev, curr) => {
+      return {
+        ...prev,
+        [curr.split(':')[0].trim()]: curr.split(':')[1].trim(),
+      };
+    }, {});
```

Comment:
While the use of `split`, `map`, `filter`, and `reduce` is correct and adheres to the checklist item about using iteration methods, the current indentation of chained methods does not follow the good example provided in the checklist. To improve readability, each chained method should start on a new line with the same indentation level.